docs/3.0rc/manage/cloud/index.mdx

```diff
@@ -1,19 +1,19 @@
 ---
 title: Prefect Cloud Overview
-description: Observe and orchestrate your workflow applications with the Prefect Cloud platform.
+description: Observe and orchestrate your workflow apps with the Prefect Cloud platform.
 ---
 
-Prefect Cloud is a hosted workflow application platform with all the capabilities of an open source Prefect server instance plus additional features, such as:
-
-- [User accounts](#user-accounts) &mdash; personal accounts for working in Prefect Cloud.
-- [Workspaces](#workspaces/) &mdash; isolated environments to organize your flows, deployments, and flow runs.
-- [Service accounts](/3.0rc/manage/cloud/manage-users/service-accounts/) &mdash; configure API access for running workers or executing flow runs on remote infrastructure.
-- [Custom role-based access controls (RBAC)](/3.0rc/manage/cloud/manage-users/manage-roles/) &mdash; assign users granular permissions to perform certain activities within an account or a workspace.
-- [Single Sign-on (SSO)](/3.0rc/manage/cloud/manage-users/configure-sso/) &mdash; authentication using your identity provider.
-- [Audit Logs](/3.0rc/manage/cloud/manage-users/audit-logs/) &mdash; a record of user activities to monitor security and compliance.
-- Collaboration &mdash; invite other people to your account.
-- Error summaries  &mdash; (enabled by Marvin AI) distill the error logs of `Failed` and `Crashed` flow runs into actionable information.
-- [Push work pools](/3.0rc/deploy/infrastructure-examples/serverless/) &mdash; run flows on your serverless infrastructure without running a worker.
+Prefect Cloud is a hosted workflow app platform with all the capabilities of an open source Prefect server instance plus additional features, such as:
```

Comment:
I think references to the Prefect application platform should stick with "application" since "application platform" is a standard term.

https://developers.google.com/style/word-list#app

> Use application in standard phrases such as application programming interface.

```suggestion
Prefect Cloud is a hosted workflow application platform with all the capabilities of an open source Prefect server instance plus additional features, such as:
```